### ISSUE TITLE
Build fails with Android build target, beta SDK 3.6.0-beta.1, on Unity 2022.3.22f1

### ISSUE THREAD

**Issue by Saberpeep** [lyuma/Av3Emulator#167]
![image](https://github.com/lyuma/Av3Emulator/assets/3688260/fc73b3b0-63da-40e5-903a-91f456f57010)

As long as Av3Emulator is installed I cannot Build and Publish to the Android build target.
Using Av3Emulator 3.4.3 installed via VCC 2.3.0-beta.3

**Comment by lyuma** [comment]
Thanks for the report!
This is the first time I've heard of an error like this.

The code on line 1062 uses `dynamic` which is a more modern C# feature. I didn't expect that some people would be using different versions of the C# runtime.

Can you please share the following:
https://docs.unity3d.com/Manual/dotnetProfileSupport.html
Show the .NET settings in **Edit > Project Settings > Player >Other settings**

And the other thing that would be helpful would be either a screenshot of your VCC package list, or at least tell us which version of VRCSDK Base and VRCSDK Avatars you have installed.

If we can't figure it out, we might have to change this code back to using plain reflection, but I was told `dynamic` was the cool and hip way of writing C# these days and it seemed to work even on my old Unity 2019 install.

**Comment by Saberpeep** [comment]
The Api Compatability Level is set to `.NET Framework`. Not sure if it changes but I made sure to check while android build target is set in the SDK.

Using SDK 3.6.0-beta.1 in Unity 2022.3.22f1. (sorry that was in the title, but not in the body of the original post).

**Comment by anatawa12** [comment]
Is the compilation error occuring for Editor build? Could you try removing the "Library" directory which is a cache folder from project?

**Comment by Saberpeep** [comment]
Weirdly I just ran into someone else who is still on 2019 having the exact same `Missing Compiler Required Member 'Microsoft.CSharp.RuntimeBinder.CSharpArgumentInfo.Create'` error coming from lyumaAv3Runtime at the same line numbers while trying to build for quest. 
Unity: 2019.4.31f1
SDK: 3.4.2
AV3: 3.4.0
So I guess its not related to the Unity nor SDK version. Unfortunately I didn't get the opportunity to troubleshoot further with them.

On my end its been a few days since I've been able to test but I can't seem to reproduce this issue anymore. I wonder if its one of those 'close unity and open it again' kind of bugs. I wouldn't blame you if you want to close this issue, since the only lead we have to go on is "unity doesn't like dynamic, but only when it wakes up on the wrong side of the bed".
